python/lsst/ts/athexapod/gcserror.py

```diff
@@ -1424,26 +1424,26 @@ def translate_error(value):
 class GCSError(Exception):
     """GCSError exception."""
 
-    def __init__(self, value, message=""):
+    def __init__(self, value: int, message: str="") -> None:
         Exception.__init__(self)
         self.val = value
         self.msg = translate_error(value)
         if message:
             self.msg += ": %r" % message
         logging.debug("GCSError: %s", self.msg)
 
-    def __str__(self):
+    def __str__(self) -> str:
         """Handle string representation."""
         return self.msg
 
-    def __repr__(self):
+    def __repr__(self) -> str:
         """Intrepeter representation."""
         return self.msg
 
-    def __eq__(self, other):
+    def __eq__(self, other: object) -> bool:
```

Comment:
Please do not use `object` as a type hint but use `typing.Any` instead, here and everywhere else. See https://docs.python.org/3.12/library/typing.html#typing.Any